src/activities/content/ContentModuleEntity.js

```diff
@@ -62,6 +62,32 @@ export class ContentModuleEntity extends Entity {
 		return this._entity && this._entity.properties && this._entity.properties.customAccentColor;
 	}
 
+	/**
+	 * @returns {string} The orgUnitId of the content-module (read-only)
+	 */
+
+	orgUnitId() {
+		return this._entity && this._entity.properties && this._entity.properties.orgUnitId;
+	}
+
+	/**
+	 * @returns {string} The moduleId of the content-module (read-only)
+	 */
+	moduleId() {
+		return this._entity && this._entity.properties && this._entity.properties.moduleId;
+	}
+
+	/**
+	 * @returns {string} Returns the endpoint for generating a summary for the module
+	 */
+	generateSummaryEndpoint() {
+		if (!this._entity || !this._entity.hasLinkByRel(Rels.Content.generateModuleSummary)) {
+			return;
+		}
+
+		return this._entity.getLinkByRel(Rels.Content.generateModuleSummary).href;
```

Comment:
`generateModuleSummary` or `generateModuleSummaryEndpoint` or `generateSummaryEndpoint`?

Reply:
Hmm yeah, I want it to be somewhat generic but the Rel is currently only in the modules domain.

Reply:
Updated to get rid of using `module` everywhere. It should be part of the entity name or prefix